<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurveSettings</class>
 <widget class="QWidget" name="CurveSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QGroupBox" name="labelGroupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Label</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="GuiUtils::FontPushButton" name="labelFontButton">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>X Offset</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="labelOffsetXSpinBox">
            <property name="minimum">
             <number>-99</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="GuiUtils::ColorPushButton" name="labelColorButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Y Offset</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QSpinBox" name="labelOffsetYSpinBox">
            <property name="minimum">
             <number>-99</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Position</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="labelPositionCombo"/>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Transparency</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QSlider" name="labelOpacitySlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Symbol</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Edge Width</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Preview</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Size</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QComboBox" name="symbolSizeCombo"/>
          </item>
          <item row="0" column="1">
           <widget class="ComDataGui::SymbolComboBox" name="symbolCombo"/>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="symbolWidthCombo"/>
          </item>
          <item row="4" column="4">
           <widget class="QSlider" name="symbolOpacitySlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Transparency</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="GuiUtils::ColorPushButton" name="symbolColorButton">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Fill Color</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Edge Color</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="GuiUtils::ColorPushButton" name="symbolEdgeColorButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Line</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Transparency</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="lineWidthCombo"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSlider" name="lineOpacitySlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="antialiasingCheck">
            <property name="text">
             <string>Anti Aliasing</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QGroupBox" name="groupBox_5">
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Dropline</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="1" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="6" column="3">
           <widget class="GuiUtils::ColorPushButton" name="droplineColorButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="GuiUtils::PenStyleBox" name="droplinePenStyleCombo"/>
          </item>
          <item row="2" column="5">
           <widget class="QComboBox" name="droplineWidthCombo"/>
          </item>
          <item row="2" column="4">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item row="6" column="4">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Transparency</string>
            </property>
           </widget>
          </item>
          <item row="6" column="5">
           <widget class="QSlider" name="droplineOpacitySlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Style</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="2">
           <widget class="QCheckBox" name="droplineShowHorizontalCheck">
            <property name="text">
             <string>Horizontal</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QCheckBox" name="droplineShowVerticalCheck">
            <property name="text">
             <string>Vertical</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1" rowspan="5">
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>Preview</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="demoLayout"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="5" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GuiUtils::ColorPushButton</class>
   <extends>QPushButton</extends>
   <header>comguiutils/colorpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>GuiUtils::FontPushButton</class>
   <extends>QPushButton</extends>
   <header>comguiutils/fontpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>GuiUtils::PenStyleBox</class>
   <extends>QComboBox</extends>
   <header>comguiutils/penstylebox.h</header>
  </customwidget>
  <customwidget>
   <class>ComDataGui::SymbolComboBox</class>
   <extends>QComboBox</extends>
   <header>comdatagui/symbolcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
